doc/onehot.md

```diff
@@ -0,0 +1,15 @@
+# One Hot Codecs
+
+ROHD HCL implements a set of one hot encoder and decoders.
+
+For example, we have an encoder [`BinaryToOneHot`](https://desmonddak.github.io/rohd-hcl/rohd_hcl/BinaryToOneHot-class.html) class, and a couple of implementations of decoder classes like [`OneHotToBinary`](https://desmonddak.github.io/rohd-hcl/rohd_hcl/OneHotToBinary-class.html) and a more performant [`BinaryToOneHot`](https://desmonddak.github.io/rohd-hcl/rohd_hcl/TreeOneHotToBinary-class.html).
```

Comment:
We should link these to the intel.github.io links, not the ones on your fork.

Also, typo I think you meant `TreeOneHotToBinary` for the 3rd link on this line

Reply:
Fixed.